src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -216,6 +216,7 @@ extern const char* key_press_mode;
 // duplicate names from hmi_request
 extern const char* limited_character_list;
 extern const char* auto_complete_text;
+extern const char* auto_complete_list;
```

Comment:
@dtrunov I am a bit confused why there is a group of fields "duplicate names from hmi_request" which is duplicated in "namespace strings". Can you tell me in which namespase should I leave the new field "namespace hmi_request" or "namespace strings".